src/transformers/testing_utils.py

```diff
@@ -2402,6 +2402,22 @@ def wrapper(*args, **kwargs):
                 subprocess.run(command, env=env, check=True, capture_output=True)
             except subprocess.CalledProcessError as e:
                 exception_message = e.stdout.decode()
+                lines = exception_message.split("\n")
+                # Add a first line with more informative information instead of just `= test session starts =`.
+                # This makes the `short test summary info` section more useful.
+                if "= test session starts =" in lines[0]:
+                    text = ""
+                    for line in lines[1:]:
+                        if line.startswith("FAILED "):
+                            text = line[len("FAILED ") :]
+                            text = "".join(text.split(" - ")[1:])
+                        elif line.startswith("=") and line.endswith("=") and " failed in " in line:
+                            break
+                        elif len(text) > 0:
+                            text += f"\n{line}"
+                    text = "(subprocess) " + text
+                    lines = [text] + lines
+                exception_message = "\n".join(lines)
```

Comment:
currently, what is shown in the (final) `short test summary info` is something like

> /transformers/src/transformers/testing_utils.py:2420: ==== test session starts ====

which is not very informative. 

And for the same test, we get two `FAILED ` in the log: one from the `subprocess` and another one is the original pytest process. This causes slack CI report script fails.

Reply:
ah got it thanks for explaining